### PR TITLE
May 2022 Steering Council community update

### PR DESCRIPTION
**Pull request by encukou**
Summarizing our notes from May.

### CODE REVIEW

File: updates/2022-05-steering-council-update.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# SC meeting summary - May 2022
+
+## 2022-05-02
+
+* SC members who attended PyCon US gave a summary of the Language Summit..
```
Comment (arhadthedev) [inline updates/2022-05-steering-council-update.md:5]:
```suggestion
* SC members who attended PyCon US gave a summary of the Language Summit.
```

File: updates/2022-05-steering-council-update.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    or not (and only point out issues in the SC review)?
+    There are differing opinions.
+    We definitely need to be careful with wording.
+* The SC discussed GIL removal.
+  Many unsolved issues remain, no one seems to want to turn the
+  proof of concept into PEPs and pull requests.
+
+## 2022-05-16
+
+Meeting was skipped. Only 2 SC members could make it..
```
Comment (arhadthedev) [inline updates/2022-05-steering-council-update.md:39]:
```suggestion
Meeting was skipped. Only 2 SC members could make it.
```

File: updates/2022-05-steering-council-update.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+* The SC discussed how to support mentoring by and among Core Developers with
+  the Executive Director.
+* The SC discussed [PEP 689](https://peps.python.org/pep-0689/) (Unstable C API tier)
+* The SC accepted [PEP 687](https://peps.python.org/pep-0687/) (Isolating modules in the standard library)
+* The SC discussed improving the bus factor situation around Python releases.
+
+## 2022-05-30
+
+* The SC discussed [PEP 689](https://peps.python.org/pep-0689/) (Unstable C API tier)
+  and decided to accept it provided that the unstable API has underscore-prefixed names.
```
Comment (gvanrossum) [inline updates/2022-05-steering-council-update.md:60]: Just curious -- the PEP is still listed as Draft status -- what happened there? (Agreed this is kind of OT for this review.)

Comment (brettcannon) [reply updates/2022-05-steering-council-update.md:60]: Disagreement over the naming scheme that halted acceptance.

Comment (encukou) [reply updates/2022-05-steering-council-update.md:60]: I thought I'd get to starting the follow-up discussion soon, but didn't yet :(
I filed https://github.com/python/peps/pull/2769/files to make the current status clear.

**Pull request merged**